README.md

```diff
@@ -143,7 +157,7 @@ Every voice is important. If you have something on your mind, create an issue or
 See the [rendered changelog](https://ansible-collections.github.io/amazon.cloud/branch/main/collections/amazon/cloud/docsite/CHANGELOG.html) or the [raw generated changelog](https://github.com/ansible-collections/amazon.cloud/tree/main/CHANGELOG.rst).
 
 
-## Communication
+## Support
 
 <!--List available communication channels. In addition to channels specific to your collection, we also recommend to use the following ones.-->
```

Comment:
I'm not sure the contents of this Support section match what is described in the template: "This section should include information about what is supported and how to get support for the collection. This can include supported versions of the collection, how to submit a support request, and any other information about how to get additional assistance."